virtualization-and-containers/material/exercise_vm_containers_text.md

```diff
@@ -184,6 +184,11 @@ We want to provision (`config.vm.provision`) the box in several steps. You can r
 - [Vagrant Homepage](https://www.vagrantup.com/)
 - [Vagrant Introduction](https://www.vagrantup.com/intro)
 - [VirtualBox Manual](https://www.virtualbox.org/manual/UserManual.html)
+- Issue with windows and vagrant: ```VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole```
+    - Solution described in this [stackoverflow thread](https://stackoverflow.com/questions/37955942/vagrant-up-vboxmanage-exe-error-vt-x-is-not-available-verr-vmx-no-vmx-code)
+    - Basically run (in powershell as admin): ```bcdedit /set hypervisorlaunchtype off```
```

Comment:
```suggestion
          1. As admin, run the following command in a PowerShell:
    
            ```powershell
            bcdedit /set hypervisorlaunchtype off
            ```
```
Again some rephrasing and fenced code block with PowerShell identifer.